objcgen/processedtypes.cs

```diff
@@ -189,17 +242,20 @@ public override void ComputeSignatures (Processor p)
 	public class ProcessedProperty: ProcessedMemberBase {
 		public PropertyInfo Property { get; private set; }
 
-		public ProcessedProperty (PropertyInfo property)
+		public ProcessedProperty (PropertyInfo property, Processor processor) : base (processor)
 		{
 			Property = property;
 		}
 
-		public override void ComputeSignatures (Processor p)
+		public override void ComputeSignatures ()
 		{
 			throw new NotImplementedException ();
 		}
 
 		public override string ToString () => Property.ToString ();
+
+		public ProcessedMethod GetMethod => new ProcessedMethod (Property.GetGetMethod (), Processor);
+		public ProcessedMethod SetMethod => new ProcessedMethod (Property.GetSetMethod (), Processor);
```

Comment:
This is also create instance of `ProcessedMethod` with null `MethodInfo` since only one (getter or setter) is required. 

I think it would be better to expose properties like:
* `bool HasGetter { get; }`
* `bool HasSetter { get; }`
* `ProcessedMethod GetMethod { get; }`
* `ProcessedMethod SetMethod { get; }`

I don't think we need to lazily create them either. If they exists we know they will be needed (and created) at some point so the `.ctor` can do all the job.